application/src/main/java/com/musicforall/util/DatabaseUtil.java

```diff
@@ -24,14 +24,12 @@ private DatabaseUtil() {
     public static void createUsersConnectionRepositoryTable(DataSource dataSource) {
         try (Statement statement = dataSource.getConnection().createStatement()) {
             statement.execute("select count(*) from UserConnection");
-            statement.execute("DROP TABLE UserConnection");
         } catch (SQLException e) {
-            LOG.error("Table UserConnection does not exist");
+            final ResourceDatabasePopulator databasePopulator = new ResourceDatabasePopulator();
```

Comment:
create UserConnection table statement should be executed only if the table doesn`t exist
also DatabaseUtil is not representative name for such kind of operations